components/Menu/Menu.scss

```diff
@@ -39,36 +42,35 @@
 @media (max-width: $breakpoint-xxl) {
   .menu {
     padding: $padding-s $padding-l;
-    .menu__menu-container {
-      .menu__list li {
-        margin-right: 0px;
-        margin: 0 $margin-xs;
-      }
+    .menu__list li {
+      margin-right: 0px;
+      margin: 0 $margin-xs;
     }
   }
 }

```

Comment:
Nie musisz definiować w ten sposób breakpointów w SCSS. Możesz go od razu deklarować pod .menu, wewnatrz tego zagnieżdżenia. Ułatwia to utrzymanie kodu bo mamy zaraz obok styli zwykłych style breakpointowe